frontend/src/components/Login.js

```diff
@@ -39,7 +39,7 @@ class Login extends Component {
 	  }
 
 	  checkUser = async () => {
-		let res = await api.get('/', { email: this.state.email, password: this.state.password})
+		let res = await api.post('/', { email: this.state.email, password: this.state.password})
 		
```

Comment:
According to REST and the CRUD standart, you should use GET to 'Read' information and POST to 'Create' stuff. Here you don't make new entry in the database, you only get answer which counts as reading. 

So it shoud be GET

Reply:
I am not sure why but my logic is that, usually i make a GET request to load the login/register page and then the submit button sends POST request for the login/register

Reply:
Fine it will be your way